docs/release-notes/NuGet-5.0-RTM.md

```diff
@@ -0,0 +1,156 @@
+
+**DCRs**
+
+* Warn of future removal of "dotnet pack project.json" - [#7928](https://github.com/NuGet/Home/issues/7928)
+ 
+* Add a deprecation warning for Gen1 credential plugin - [#7819](https://github.com/NuGet/Home/issues/7819)
+ 
+* Signing: Enabled Repo to require client verification of every package as repo signed -- via RepositorySignatures/5.0.0 resource - [#7759](https://github.com/NuGet/Home/issues/7759)
+
+* limit http request number per source through NuGet.Config - [#4538](https://github.com/NuGet/Home/issues/4538)
+
+* NuGet should target Net472 (to help Cleanup the 16.0 build of the VSIX) - [#7143](https://github.com/NuGet/Home/issues/7143)
+
+* PMC: Remove OpenPackagePage command - [#7384](https://github.com/NuGet/Home/issues/7384)
+
+* Make NetCoreApp 3.0 map to NetStandard 2.1 - [#7762](https://github.com/NuGet/Home/issues/7762)
+
+* Add netstandard2.0 support to NuGet.* packages - [#6516](https://github.com/NuGet/Home/issues/6516)
+
+* Allow package authors to define build assets transitive behavior - [#6091](https://github.com/NuGet/Home/issues/6091)
+
+* Enable restore in VS to succeed if a project is not part of solution or is not loaded, but has previously been restored - [#5820](https://github.com/NuGet/Home/issues/5820)
+
+* NuGet 5.0 assemblies to require .NET 4.7.2 (via TFM change) - [#7510](https://github.com/NuGet/Home/issues/7510)
+
+* NuGetLicenseData from NuGet.Packaging should be a public type. Update license metadata ingested from spdx. - [#7471](https://github.com/NuGet/Home/issues/7471)
+
+* Remove obsolete Settings APIs - [#7294](https://github.com/NuGet/Home/issues/7294)
+
+* Workaround restore timeouts on systems with 1 cpu - [#6742](https://github.com/NuGet/Home/issues/6742)
+
+* NuGet prefers NTLM auth even if there are credentials in NuGet.config - add config option to filter auth types for credentials - [#5286](https://github.com/NuGet/Home/issues/5286)
+
+* Enable EmbedInteropTypes for PackageReference (matching Packages.Config capability) - [#2365](https://github.com/NuGet/Home/issues/2365)
+
+## Known issues
+
+### Packages in FallbackFolders installed by .NET Core SDK are custom installed, and fail signature validation. - [#7414](https://github.com/NuGet/Home/issues/7414)
+#### Issue
+When using dotnet.exe 2.x to restore a project that multi-targets netcoreapp 1.x and netcoreapp 2.x, the fallback folder is treated as a file feed. This means, when restoring, NuGet will pick the package from the fallback folder and try to install it into the global packages folder and do the usual signing validation which fails.<br>
+#### Workaround
+Disable the usage of the fallback folder by setting the `RestoreAdditionalProjectSources` to nothing. `<RestoreAdditionalProjectSources/>` Use this with caution as it will cause a lot of packages to be downloaded from NuGet.org which otherwise would be have been restored from the fallback folder.
```

Comment:
This looks a little weird on the rendered page: https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.0-rtm?branch=pr-en-us-1378#workaround

```suggestion
Disable the usage of the fallback folder by setting the `RestoreAdditionalProjectSources` to nothing:

`<RestoreAdditionalProjectSources />`

Use this with caution as packages that would be restored from the fallback folder will now be downloaded from NuGet.org.
```

Reply:
Do we have documentation for the `RestoreAdditionalProjectSources` property that we could link to?

Reply:
we have an issue to create those docs (#1358) - i've added your suggestion to that doc issue to do this once that doc exists.